input/otlp/traces.go

```diff
@@ -495,7 +501,12 @@ func TranslateTransaction(
 		event.Url = modelpb.ParseURL(httpURL, httpHost, httpScheme)
 	}
 	if isMessaging {
-		event.Transaction.Message = &message
+		// Overwrite existing event.Transaction.Message
```

Comment:
Moved this code back here. It is important to overwrite existing event.Transaction.Message to avoid breaking otel bridge test when event.Transaction.Message and the otel attribute exist at the same time.
```
=== RUN   TestDecodeMapToTransactionModel/otel-bridge/messaging
    transaction_test.go:559: 
                Error Trace:    /home/carson/projects/apm-data/input/elasticapm/internal/modeldecoder/v2/transaction_test.go:559
                Error:          Not equal: 
                                expected: &modelpb.Message{state:impl.MessageState{NoUnkeyedLiterals:pragma.NoUnkeyedLiterals{}, DoNotCompare:pragma.DoNotCompare{}, DoNotCopy:pragma.DoNotCopy{}, atomicMessageInfo:(*impl.MessageInfo)(nil)}, sizeCache:0, unknownFields:[]uint8(nil), Body:"", Headers:[]*modelpb.HTTPHeader(nil), AgeMillis:(*uint64)(nil), QueueName:"myQueue", RoutingKey:""}
                                actual  : &modelpb.Message{state:impl.MessageState{NoUnkeyedLiterals:pragma.NoUnkeyedLiterals{}, DoNotCompare:pragma.DoNotCompare{}, DoNotCopy:pragma.DoNotCopy{}, atomicMessageInfo:(*impl.MessageInfo)(nil)}, sizeCache:0, unknownFields:[]uint8(nil), Body:"init", Headers:[]*modelpb.HTTPHeader{(*modelpb.HTTPHeader)(0xc00040aa50)}, AgeMillis:(*uint64)(0xc000327058), QueueName:"myQueue", RoutingKey:"init"}
```

Reply:
I don't see how the message instance would already be non-nil at this point for otel data, so it should be fine.